creator-node/src/snapbackSM/snapbackSM.js

```diff
@@ -187,7 +213,7 @@ class SnapbackSM {
    */
   async init() {
     // Empty all queues to minimize memory consumption
-    await this.stateMachineQueue.empty()
+    await this.stateMachineQueue.obliterate({ force: true })
```

Comment:
ah - just remembered. this won't clear state from state machine queues with legacy naming format since its on the current instance. it will ensure no prior state carried over for current queue which still seems valuable
but clearly clearing legacy queue state does not fix stall issue
just making sure you've confirmed queue runs normally after this obliterate cmd right

Reply:
ah interesting. I did confirm the queue ran normally on each machine tested. issues didn't start until after at least 50 or 100 new jobs, and the visualization shows that there aren't a bunch of other jobs stuck in the queue